src/main/java/com/jcabi/log/ControlSequenceIndicatorFormatted.java

```diff
@@ -52,7 +52,7 @@ public ControlSequenceIndicatorFormatted(final String pat) {
 
     @Override
     public final String format() {
-        return String.format(this.pattern, "\u001b[");
+        return String.format(this.pattern, "\u001b\\[");
```

Comment:
Same as aboce, since ``\u001b[`` will be part of a regex, we need to escape the ``[``, which has special meaning.